private/buf/bufformat/testdata/proto3/all/v1/all.proto

```diff
@@ -69,7 +69,9 @@ message One {
   int64 one_one = 1 [ (custom.field_option) = true];
   Two one_two = 2[
   (custom.field_option) = false,
-  (custom.field_thing_option) = { foo: 6}
+  (custom.field_thing_option) = {
+     foo: 6, // comment after foo
```

Comment:
Do we need the trailing comma?

Reply:
This is the input file. If you look in the golden version of this file, the comma is removed.

As to why the comma is present in the input -- it's to make sure we don't lose the comment, which is attached to the comma which we _are_ removing.